docs/Tutorials/Send-private-transaction.md

```diff
@@ -0,0 +1,111 @@
+---
+description: Sending private transactions
+---
+
+# Send private transactions
+
+## Prerequisites
+
+* [Privacy-enabled network as configured in tutorial](Create-Privacy-enabled-network.md).
+
+## Steps
+
+Listed on the right-hand side of the page are the steps to create a private contract, deploy the contract,
+and send a private transaction.
+
+## 1. Create private contract
+
+In the `Node-0` directory, copy and paste the following to a file called `private-config.js`. Replace
+the placeholder for `privateFor` with the `tessera1.pub` key.
```

Comment:
Add some detail to prevent user to put `tessera1.pub` or the file path as this value.
```suggestion
the `<Tessera 1 public key>` placeholder for `privateFor` with the base64 content of the `tessera1.pub` key file. For example: `AxqQUAXxuDEqzjjTPuF9hGRbTlpAiWN6xVHWwp5ki2A=`.
```

Reply:
beautifying the script and adding line numbers and highlight the one to change would also be helpful. The sentence could be : 

> [...]the `<Tessera 1 public key>` placeholder (at line 52) for `privateFor` with[...]

With the code formated using the line numbers and the highlight options:

```
```javascript hl_lines="52" linenums="1"
a = eth.accounts[0]
web3.eth.defaultAccount = a;

// abi and bytecode generated from simplestorage.sol:
// > solcjs --bin --abi simplestorage.sol
var abi = [{
    "constant": true,
    "inputs": [],
    "name": "storedData",
    "outputs": [{
        "name": "",
        "type": "uint256"
    }],
    "payable": false,
    "type": "function"
}, {
    "constant": false,
    "inputs": [{
        "name": "x",
        "type": "uint256"
    }],
    "name": "set",
    "outputs": [],
    "payable": false,
    "type": "function"
}, {
    "constant": true,
    "inputs": [],
    "name": "get",
    "outputs": [{
        "name": "retVal",
        "type": "uint256"
    }],
    "payable": false,
    "type": "function"
}, {
    "inputs": [{
        "name": "initVal",
        "type": "uint256"
    }],
    "payable": false,
    "type": "constructor"
}];

var bytecode = "0x6060604052341561000f57600080fd5b604051602080610149833981016040528080519060200190919050505b806000819055505b505b610104806100456000396000f30060606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680632a1afcd914605157806360fe47b11460775780636d4ce63c146097575b600080fd5b3415605b57600080fd5b606160bd565b6040518082815260200191505060405180910390f35b3415608157600080fd5b6095600480803590602001909190505060c3565b005b341560a157600080fd5b60a760ce565b6040518082815260200191505060405180910390f35b60005481565b806000819055505b50565b6000805490505b905600a165627a7a72305820d5851baab720bba574474de3d09dbeaabc674a15f4dd93b974908476542c23f00029";

var simpleContract = web3.eth.contract(abi);
var simple = simpleContract.new(42, {
    from: web3.eth.accounts[0],
    data: bytecode,
    gas: 0x47b760,
    privateFor: ["<Tessera 1 public key>"]
}, function(e, contract) {
    if (e) {
        console.log("err creating contract", e);
    } else {
        if (!contract.address) {
            console.log("Contract transaction send: TransactionHash: " + contract.transactionHash + " waiting to be mined...");
        } else {
            console.log("Contract mined! Address: " + contract.address);
            console.log(contract);
        }
    }
});
```
```

Reply:
Implemented both suggestions